client/state/selectors/test/get-application-passwords.js

```diff
@@ -0,0 +1,45 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { getApplicationPasswords } from 'state/selectors';
+
+describe( 'getApplicationPasswords()', () => {
+	test( 'should return application passwords of the current user', () => {
+		const appPasswords = [
+			{
+				ID: 12345,
+				name: 'Example',
+				value: '2018-01-01T00:00:00+00:00',
+			},
+			{
+				ID: 23456,
+				name: 'Test',
+				value: '2018-02-01T08:10:00+00:00',
+			},
+		];
+		const state = {
+			applicationPasswords: {
+				items: appPasswords,
+			},
+		};
+		const result = getApplicationPasswords( state );
+		expect( result ).toEqual( appPasswords );
+	} );
+
+	test( 'should return an empty array if no application passwords exist', () => {
+		const state = {
+			applicationPasswords: {
+				items: [],
+			},
+		};
+		const result = getApplicationPasswords( state );
+		expect( result ).toEqual( [] );
```

Comment:
Does this test add any value? It's testing the same behavior as the previous test, isn't it?

Reply:
You're right - removed in https://github.com/Automattic/wp-calypso/pull/23906/commits/787c87f0b9618bfa9f7277783108a0aeb66b4b09.